internal/mode/static/state/graph/httproute_test.go

```diff
@@ -330,6 +341,8 @@ func TestBuildRoute(t *testing.T) {
 	const (
 		invalidPath             = "/invalid"
 		invalidRedirectHostname = "invalid.example.com"
+		emptyPathType           = "/empty-type"
```

Comment:
better to move these new constants to the global scope so that createHTTPRoute can also use them